examples/docs_beta_snippets/docs_beta_snippets/guides/data-modeling/asset-factories/simple-yaml-asset-factory.py

```diff
@@ -33,5 +37,5 @@ def load_etl_jobs_from_yaml(yaml_path: str) -> dg.Definitions:
     return dg.Definitions.merge(*defs)
 
 
-defs = load_etl_jobs_from_yaml("etl_jobs.yaml")
+defs = load_etl_jobs_from_yaml(str(Path(__file__).parent / "etl_jobs.yaml"))
```

Comment:
hoping we can get rid of this in all the examples and replace it with a chdir in the test case so the code samples are simpler

Reply:
I think this is a decent compromise, what do you think?

https://github.com/dagster-io/dagster/pull/23909/commits/3234f3dc82a98d2828d8a8d456cb82c684f67f79

Reply:
still think it's pretty noisy to have as a code sample in the guides